plugins/modules/ipauser.py

```diff
@@ -1449,6 +1449,10 @@ def main():
                                 del args["userpassword"]
                             if "random" in args:
                                 del args["random"]
+                        # if using "random:false" password should not be
```

Comment:
Wouldn't an else be good here? It would match the commit message.

Reply:
I don't think it would be useful, as the else part would be a "no-op".

Maybe changing the comment, but I did not find a better way to rewrite. Any suggestion is welcome.